src/common.speech/RequestSession.ts

```diff
@@ -186,7 +191,7 @@ export class RequestSession {
     }
 
     public onStopRecognizing(): void {
-        this.privIsRecognizing = false;
+        this.onComplete();
```

Comment:
it seems the onComplete is called when turn ends. etc. Add this seems not hurting. Does it necessary?

Reply:
OnComplete also stops the audio pump when it detaches the audio node.

---
In reply to: [324360733](https://github.com/microsoft/cognitive-services-speech-sdk-js/pull/83#discussion_r324360733) [](ancestors = 324360733)